main.py

```diff
@@ -1,147 +1,141 @@
     return update_energy_buy(engine, omie_temp_dir, verbose, dry_run)
 
+
 def main_update_neuroenergia(verbose, dry_run):
-    engine = None if dry_run else create_engine(local_db['dbapi'])
-    neuro_dir = directories['NEUROENERGIA_TEMP']
-    return update_neuroenergia(engine,neuro_dir,verbose,dry_run)
+    engine = None if dry_run else create_engine(local_db["dbapi"])
+    neuro_dir = directories["NEUROENERGIA_TEMP"]
+    return update_neuroenergia(engine, neuro_dir, verbose, dry_run)
+
 
 def main_update_conversations(verbose, dry_run):
-    engine = create_engine(local_db['dbapi'])
-    hs_app_id = helpscout_api['app_id']
-    hs_app_secret = helpscout_api['app_secret']
+    engine = create_engine(local_db["dbapi"])
+    hs_app_id = helpscout_api["app_id"]
+    hs_app_secret = helpscout_api["app_secret"]
     return update_hs_conversations(engine, hs_app_id, hs_app_secret, verbose, dry_run)
 
+
 def main_pipe_hourly_energy_budget(verbose, dry_run):
-    engine = None if dry_run else create_engine(local_db['dbapi'])
+    engine = None if dry_run else create_engine(local_db["dbapi"])
     return pipe_hourly_energy_budget(engine, verbose)
 
+
 def main_pipe_omie_garantia(verbose, dry_run):
-    engine = None if dry_run else create_engine(local_db['dbapi'])
+    engine = None if dry_run else create_engine(local_db["dbapi"])
     return pipe_omie_garantia(engine)
 
 
 function_list = {
-    'meff_update_closing_prices_day': main_update_closing_prices_day,
-    'meff_update_closing_prices_month': main_update_closing_prices_month,
-    'omie_get_historical_hour_price': main_get_historical_hour_price,
-    'omie_update_latest_hour_price': main_update_omie_latest_hour_price,
-    'omie_update_historical_hour_price': main_update_historical_hour_price,
-    'omie_get_historical_energy_buy' : main_get_historical_energy_buy,
-    'omie_update_energy_buy' : main_update_energy_buy,
-    'neuro_update_energy_prediction' : main_update_neuroenergia,
-    'hs_update_conversations' : main_update_conversations,
-    'pipe_hourly_energy_budget' : main_pipe_hourly_energy_budget,
-    'pipe_omie_garantia' : main_pipe_omie_garantia,
+    "meff_update_closing_prices_day": main_update_closing_prices_day,
+    "meff_update_closing_prices_month": main_update_closing_prices_month,
+    "omie_get_historical_hour_price": main_get_historical_hour_price,
+    "omie_update_latest_hour_price": main_update_omie_latest_hour_price,
+    "omie_update_historical_hour_price": main_update_historical_hour_price,
+    "omie_get_historical_energy_buy": main_get_historical_energy_buy,
+    "omie_update_energy_buy": main_update_energy_buy,
+    "neuro_update_energy_prediction": main_update_neuroenergia,
+    "hs_update_conversations": main_update_conversations,
+    "pipe_hourly_energy_budget": main_pipe_hourly_energy_budget,
+    "pipe_omie_garantia": main_pipe_omie_garantia,
 }
 
-# TODO use logging instead of -v
+
 @click.command()
```

Comment:
In new projects we're using typer instead of click directly. This PR might be a good timing to make the swap.

Reply:
I'd rather update this on another PR, if you don't mind. I'll create a new one.